operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerWrapper.java

```diff
@@ -32,6 +33,27 @@ public InformerWrapper(SharedIndexInformer<T> informer) {
   public void start() throws OperatorException {
     try {
       informer.run();
+
+      // register stopped handler if we have one defined
+      ConfigurationServiceProvider.instance().getInformerStoppedHandler()
+          .ifPresent(ish -> {
+            final var stopped = informer.stopped();
+            if (stopped != null) {
```

Comment:
Should we have here the null check?

Reply:
It should never be `null` but I've seen cases where it was (though I think this was due to a race condition that I fixed)… In any case, it doesn't hurt to have it and we have an explicit error in case it's ever `null`.